sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2572,7 +2572,7 @@ object functions {
   }
 
   /**
-   * Returns the current date in the UTC time zone as a date column.
+   * Returns the current date as a date column.
```

Comment:
do we need to change the python and R functions?

Reply:
No, the PR https://github.com/apache/spark/pull/24185 did touch them. So, comments for the functions don't mention UTC time zone.